apps/native-component-list/src/screens/PickerScreen.tsx

```diff
@@ -56,7 +72,9 @@ PickerScreen.navigationOptions = {
   title: 'Picker',
 };
 
-function GenericPicker(props: Partial<React.ComponentProps<typeof Picker>>) {
+function GenericPicker(
+  props: Partial<React.ComponentProps<typeof Picker>> & { children?: React.ReactNode }
```

Comment:
What do you think about using `PropsWithChildren` in there?

Reply:
thank you I will update - I always seem to type children wrong, I should brush up on my react types!